drupal/acquia_connector/sa-contrib-2019-014.yaml

```diff
@@ -0,0 +1,6 @@
+link: 'https://www.drupal.org/sa-contrib-2019-014'
+title: 'Acquia Connector - Moderately critical - Access bypass - SA-CONTRIB-2019-014'
+branches:
+    1.16.x: { time: '2019-02-06 16:38:32', versions: ['>=1.0.0', '<1.16.0'] }
```

Comment:
please dump the branch details in expanded format, as done for formatting the advisories edited manually (meaning, increase the depth of the expanded dumping by 1).

If you use the symfony/yaml dumper, that's controlled by the second argument of the method.

Reply:
👍 done